doctr/travis.py

```diff
@@ -186,3 +188,23 @@ def push_docs():
         run(['git', 'push', '-q', 'origin_token', 'gh-pages'])
     else:
         print("The docs have not changed. Not updating")
+
+
+DOCS_REGEX = re.compile('([\w.]*)\/conf\.py')
+
+def find_doc_dir():
+    """
+    Locate the local ``Sphinx`` directory
+
+    Searches recursively from the root directory for a ``conf.py`` file and
+    then backs out the name of the docs folder from there.
+
+    Output is printed for purposes of capture by bash on Travis
+    """
+    for conf_loc in glob.iglob('**/conf.py', recursive=True):
+        doc_loc = re.search(DOCS_REGEX, conf_loc)
+        if doc_loc:
+            doc_loc = doc_loc.group(1)
+            #print this to grab it in a BASH envvar
+            print(doc_loc)
```

Comment:
I actually don't really see much value in getting this automatically just to build the docs. The individual project .travis.yml will know where its docs are. The real thing you need to find is the `_build` directory.